gnoblog-cli/cmd/util.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/adrg/frontmatter"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+func parsePost(reader io.Reader) (*post, error) {
```

Comment:
The function belongs to `post.go`, not `util.go`.

Reply:
Moved it: [3d32a97](https://github.com/gnolang/blog/pull/63/commits/3d32a97c5c016146983686cfd8174d7f419175e7)